src/app/app.component.ts

```diff
@@ -1,22 +1,46 @@
-import { Component } from '@angular/core';
-import { Platform } from 'ionic-angular';
+import { Component, ViewChild } from '@angular/core';
+import { Nav, Platform } from 'ionic-angular';
 import { StatusBar } from '@ionic-native/status-bar';
 import { SplashScreen } from '@ionic-native/splash-screen';
 
-import { TabsPage } from '../pages/tabs/tabs';
+import { WificallPage } from '../pages/wificall/wificall';
+import { AdminPage } from '../pages/admin/admin';
+
+type page = {title: string, component: any};
 
 @Component({
   templateUrl: 'app.html'
 })
 export class MyApp {
```

Comment:
@tokland Can we maybe rename it to Fire-WiFi calling? @ifoche What is actually the name?